Modules/OptionHolder.cs

```diff
@@ -3821,6 +3835,38 @@ public static void SetupRoleOptions(int id, TabGroup tab, CustomRoles role, Cust
         CustomRoleSpawnChances.Add(role, spawnOption);
         CustomRoleCounts.Add(role, countOption);
     }
+    public static void SetupGhostRoleOptions(int id, TabGroup tab, CustomRoles role, CustomGameMode customGameMode = CustomGameMode.Standard, bool zeroOne = false)
+    {
+        var spawnOption = StringOptionItem.Create(id, role.ToString(), zeroOne ? ratesZeroOne : ratesToggle, 0, tab, false).SetColor(Utils.GetRoleColor(role))
+            .SetHeader(true)
+            .SetGameMode(customGameMode) as StringOptionItem;
+        var countOption = IntegerOptionItem.Create(id + 1, "Maximum", new(1, 15, 1), 1, tab, false)
+         .SetParent(spawnOption)
+            .SetValueFormat(OptionFormat.Players)
+            .SetGameMode(customGameMode);
+
+        if(CustomGhostRoleCounts.ContainsKey(role))
+            CustomRoleCounts.Remove(role);
+
+
+        //CustomRoleSpawnChances.Add(role, spawnOption);
+        CustomRoleCounts.Add(role, countOption);
+        //if (role.RoleExist())
```

Comment:
So basically find a check for https://i.gyazo.com/cad823b1cb3a003066eaafbc690ce7a6.gif here.

Reply:
I might just make another string option which is "Enabled - Disabled" 😂